### ISSUE TITLE
Xcode 9 support

### ISSUE THREAD

**Issue by robertwijas** [asmallteapot/cocoapods-playgrounds#52]
Current version doesn't work with Xcode 9.

It fails with:

```
Errno::ENOENT - No such file or directory - /Applications/Xcode-beta.app/Contents/Developer/Library/Xcode/Templates/File Templates/Source/Playground with Platform Choice.xctemplate
```

Looks like Xcode 9 comes with a different Playground templates.

**Comment by doozMen** [comment]
I am having the same issue. Full output is

### Error

```
Errno::ENOENT - No such file or directory - /Applications/Xcode.app/Contents/Developer/Library/Xcode/Templates/File Templates/Source/Playground with Platform Choice.xctemplate
/Library/Ruby/Gems/2.0.0/gems/cocoapods-playgrounds-1.2.2/lib/cocoapods-playgrounds/generate.rb:43:in `open'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-playgrounds-1.2.2/lib/cocoapods-playgrounds/generate.rb:43:in `foreach'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-playgrounds-1.2.2/lib/cocoapods-playgrounds/generate.rb:43:in `dir_for_platform'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-playgrounds-1.2.2/lib/cocoapods-playgrounds/generate.rb:10:in `initialize'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-playgrounds-1.2.2/lib/cocoapods-playgrounds/workspace.rb:26:in `new'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-playgrounds-1.2.2/lib/cocoapods-playgrounds/workspace.rb:26:in `block in generate'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-playgrounds-1.2.2/lib/cocoapods-playgrounds/workspace.rb:23:in `chdir'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-playgrounds-1.2.2/lib/cocoapods-playgrounds/workspace.rb:23:in `generate'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-playgrounds-1.2.2/lib/cocoapods-playgrounds/command/playgrounds.rb:46:in `run'
/Library/Ruby/Gems/2.0.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.3.1/lib/cocoapods/command.rb:52:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.3.1/bin/pod:55:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'
```

**Comment by mprogers** [comment]
Oh, good, I'm not alone. But since this was posted a month ago, I don't take this to be a good sign. Too bad, I was really looking forward to be able to get this to work.

**Comment by tennantje** [comment]
Same issue

**Comment by unlitdev** [comment]
Same problem

**Comment by noppefoxwolf** [comment]
Same issue. +1 Xcode9.1

**Comment by bsrz** [comment]
I have created a lane to use with fastlane that achieves this with Carthage.
Feel free to use it: https://github.com/bsarrazin/lanes

Run `$ bundle exec fastlane playground docs:true` to see how it work

**Comment by benzai** [comment]
I have the same issue. Xcode 9.2

**Comment by Laptopmini** [comment]
Seems like someone has already started working on it:

https://github.com/noppefoxwolf/ThisCouldBeUsButYouPlaying/commits/ac5dd62cfa90d87117d504e6674c7e4de72a9b7e

```
git clone https://github.com/noppefoxwolf/ThisCouldBeUsButYouPlaying.git
git checkout ac5dd62cfa90d87117d504e6674c7e4de72a9b7e
```

**Comment by JadianZheng** [comment]
@Laptopmini And then, how to apply it?

**Comment by postmechanical** [comment]
The approach in the linked commit doesn't look very safe. It bundles the template that has been removed in Xcode 9.x into the plugin itself instead of using a supported Xcode 9.x bundled template.

**Comment by asmallteapot** [comment]
Fixed in #56. A new release is coming soon!

**Comment by Sajjon** [comment]
I've managed to get it working in Xcode 10.1 by downloading `Playground with Platform Choice.xctemplate` from [this Github repo](https://github.com/youngshook/XcodeFileTemplates) and placing it in `/Applications/Xcode.app/Contents/Developer/Library/Xcode/Templates/File Templates/Source`

Then in the `My.playground` replace the first lines with:

```swift
import PlaygroundSupport
PlaygroundPage.current.needsIndefiniteExecution = true
```
